<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Exporter V0.2</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="prevLabels" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QLabel" name="l_src">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="l_target">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>25</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="details" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Klien</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>le_klien</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_klien">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>0</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>25</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>le_namafile</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_pages">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>75</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bahan</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>le_bahan</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_bahan">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>50</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>75</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pages</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>le_pages</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_namafile">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>25</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_bb">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>77</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>BB</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="gbLaminasi">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>100</y>
       <width>290</width>
       <height>54</height>
      </rect>
     </property>
     <property name="title">
      <string>       Laminasi</string>
     </property>
     <widget class="QComboBox" name="cbHotCold">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>20</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Bebas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Panas</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dingin</string>
       </property>
      </item>
     </widget>
     <widget class="QRadioButton" name="rbDoff">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>23</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DOFF</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbGlossy">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>23</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>GLOSSY</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbLaminasi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-1</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbPotong">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>160</y>
       <width>290</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string notr="true">       Potongan</string>
     </property>
     <widget class="QCheckBox" name="cb_plong">
      <property name="geometry">
       <rect>
        <x>121</x>
        <y>23</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PLONG</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_cutting">
      <property name="geometry">
       <rect>
        <x>212</x>
        <y>23</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>CUTTING</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-1</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_potong">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>23</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>POTONG</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="le_qty">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>75</y>
       <width>76</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>76</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>QTY</string>
     </property>
     <property name="buddy">
      <cstring>le_qty</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>220</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Keterangan Tambahan </string>
     </property>
     <property name="buddy">
      <cstring>le_keterangan</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_keterangan">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>220</y>
       <width>159</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="le_exportfolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="b_browse">
    <property name="geometry">
     <rect>
      <x>257</x>
      <y>85</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>373</y>
      <width>301</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="stackedWidgetPage1" native="true">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="b_export">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>EXPORT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="stackedWidgetPage2" native="true">
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QProgressBar" name="pb_status">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionView_Log"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPDF_Settings"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionView_Log">
   <property name="text">
    <string>View Log</string>
   </property>
  </action>
  <action name="actionPDF_Settings">
   <property name="text">
    <string>PDF Settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_exportfolder</tabstop>
  <tabstop>b_browse</tabstop>
  <tabstop>le_klien</tabstop>
  <tabstop>le_namafile</tabstop>
  <tabstop>le_bahan</tabstop>
  <tabstop>le_pages</tabstop>
  <tabstop>cb_bb</tabstop>
  <tabstop>le_qty</tabstop>
  <tabstop>cbLaminasi</tabstop>
  <tabstop>cbHotCold</tabstop>
  <tabstop>rbDoff</tabstop>
  <tabstop>rbGlossy</tabstop>
  <tabstop>checkBox_7</tabstop>
  <tabstop>cb_potong</tabstop>
  <tabstop>cb_plong</tabstop>
  <tabstop>cb_cutting</tabstop>
  <tabstop>le_keterangan</tabstop>
  <tabstop>b_export</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
